inbm-v5.md

```diff
@@ -0,0 +1,329 @@
+
+- **Story 1.2:** Installer/Uninstaller & .deb Package  
+  - Install/uninstall Turtle Creek daemon and inbc CLI using .deb packages  
+  - *single* .deb package created, installer shell script works, uninstall shell script works.
+
+- **Story 1.3:** Turtle Creek Daemon as a systemd Service  
+  - Turtle Creek daemon to run automatically on system boot  
+  - systemd service file created, daemon auto-starts, logs configured.
+
+- **Story 1.4:** inbc <-> Daemon Communication via UNIX Socket  
+  - inbc CLI communicates with the Turtle Creek daemon through a UNIX socket  
+  - UNIX socket communication established, error handling for invalid commands.
+
+- **Story 1.5:** provision-tc Skeleton & Service Enablement  
+  - Run a `provision-tc` command that enables and starts the Turtle Creek daemon  
+  - `provision-tc` script/command enabling daemon, logging actions. Maintain compatibility with Turtle Creek v4.
+
+- **Story 1.6:** CI/CD Setup with Scans & Integration Tests  
+  - Automated builds/tests to run in Jenkins with security scans  
+  - Jenkins pipeline configured, scanning tools integrated, basic integration test set up.
+
+### Epic 2: Security
+
+**Goal:** Implement foundational security features such as TPM/LUKS for startup, AppArmor profile.
+
+- **Story 2.1:** TPM/LUKS Setup (Reuse v4 Scripts)  
+  - System uses TPM/LUKS encryption at startup  
+  - TPM/LUKS scripts integrated: can borrow from Turtle Creek v4; use same scheme/directory layout.
+
+- **Story 2.2:** AppArmor Profile  
+  - AppArmor profile for the Turtle Creek daemon  
+  - AppArmor profile enforced when Turtle Creek is installed
+
+### Epic 3: Basic SOTA
+
+**Goal:** Implement basic SOTA updates for Ubuntu and Tiber, with optional rollback/health checks.
+
+- **Story 3.1:** Ubuntu SOTA Without Rollback/Health Check  
+  - Deploy SOTA updates via inbc without rollback or health checks  
+  - Manifest format defined, update applied, success/failure logged.
+
+- **Story 3.2:** Ubuntu SOTA with Rollback/Health Check  
+  - Rollback/health checks on system reboot  
+  - Health-check implemented, rollback logic added, reboot scenarios tested.
+
+- **Story 3.3:** Tiber A/B Updates Initially  
+  - Deploy A/B updates  
+  - Download, update, and rollback on failure should function properly (use Turtle Creek v4 as reference).
+
+### Epic 4: Clouds
+
+**Goal:** Connect to cloud backends (Azure or INBS/UDM) for SOTA updates, configured via `adapter.cfg`.
+
+- **Story 4.1:** Azure SOTA via Manifest  
+  - System connects to Azure for SOTA using a manifest  
+  - Connect to Azure working; Turtle Creek logs events to Azure; Turtle Creek responds to SOTA manifest properly and reconnects on reboot  
+
+- **Story 4.2:** INBS/UDM SOTA via gRPC  
+  - System connects to INBS/UDM over gRPC  
+  - `adapter.cfg` for INBS, Turtle Creek should respond to pings and to SOTA requests; only need to support 'immediate' requests (no scheduling needed); should send job status when done.
```

Comment:
Does this mean that the dispatcher won't have a database like the current one? If it doesn't connect with INBS/UDM, it doesn't have the ability to schedule OTA updates.